pom.xml

```diff
@@ -65,9 +65,12 @@
     <!-- Whether the build should fail if findbugs finds any error. -->
     <!-- It is strongly encouraged to leave it as true. Use false with care only in transient situations. -->
     <findbugs.failOnError>true</findbugs.failOnError>
-    <!-- Defines a default FindBugs threshold. -->
-    <!-- In Plugin POM 2.x the default value was Medium, but it was not enough to discover some null handling cases -->
-    <findbugs.threshold>Low</findbugs.threshold>
+    <!-- Defines a FindBugs threshold. Use "Low" to discover low-priority bugs.
+         Hint: FindBugs considers some real NPE risks in Jenkins as low-priority issues, it is recommended to enable it in plugins.
+      -->
+    <findbugs.threshold>Medium</findbugs.threshold>
+    <!-- Defines a FindBugs effor. Use "Max" to maximize the scan depth -->
+    <findbugs.effort>default</findbugs.effort>
```

Comment:
The only reason I am
Ok with these properties - rather than plugins just *shock horror* configuring the standard Maven way - is because it allows CLI override which enables easier exploration of the potential impact of changing these levels